File: Automorphic Number/file.c
#include<stdio.h>

int automorphic(int n){
    
    int sq=n*n;
    int flag=1;
    while(n>0){
        
        if(sq%10!=n%10){
            flag=0;
        }
        n/=10;
        sq/=10;
    }
    return flag;
}

int main()
{
    int num;
    scanf("%d",&num);
    int flag=automorphic(num);
    flag==1?printf("Yes it's an Automorphic Number.") : printf("No, it's not an Automorphic Number.");
}

File: Automorphic Number/file.py
def automorphic(n):
    sq=n*n
    while(n>0):
        if(n%10!=sq%10):
            return False
            break
        n=n//10
        sq=sq//10
    return True

num=int(input())
x=automorphic(num)
print("Yes it's an Automorphic Number.") if(x==True) else print("No it's not an Automorphic Number.")
